golangci.yaml

```diff
@@ -200,14 +200,9 @@ linters-settings:
     disable:
       - shadow
     disable-all: false
-  depguard:
-    list-type: blacklist
-    include-go-root: false
-    packages:
-      - github.com/sirupsen/logrus
-    packages-with-error-message:
-      # specify an error message to output when a blacklisted package is used
-      - github.com/sirupsen/logrus: "logging is allowed only by logutils.Log"
+  # depguard:
```

Comment:
This one I removed because I understood from the issue it is not working at the moment, but I am not used to work with Go, please confirm that I did not remove a CI check

Reply:
I'm not sure about the background of this check, but looks like keeping it has no harm.